src/main/java/com/database/operations/CompanyRepository.java

```diff
@@ -0,0 +1,85 @@
+package com.database.operations;
+
+import com.database.data.Company;
+import com.database.data.DBCloseConnect;
+import com.database.data.JdbcConnectionUtil;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyRepository {
+
+    // Qwery1 - SELECT_ALL, Qwery2 - SELECT_ID , Qwery3 - INSERT;  Qwery4 - DELETE;  Qwery5 - UPDATE;
+    private static final String Qwery1 = "SELECT * FROM companies";
+    private static final String Qwery2 = "SELECT * FROM companies WHERE id = ?";
+    private static final String Qwery3 = "INSERT INTO companies(name, address) VALUES(?, ?)";
+    private static final String Qwery4 = "DELETE FROM companies WHERE id = ?";
+    private static final String Qwery5 = "UPDATE companies SET name = ?, address = ? WHERE id = ?";
+
+    public List<Company> selectAll() throws SQLException {
+        Connection connection = JdbcConnectionUtil.getConnection();
+        if (connection == null) throw new AssertionError();
```

Comment:
1. Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.
 https://mate-academy.github.io/style-guides/java/java.html#s4.1.1-braces-always-used
2. We really should throw an object of error or we work with exceptions.
![image](https://user-images.githubusercontent.com/20267148/55960208-f7828a80-5c74-11e9-9021-22948ca317b9.png)
Do you know what is the difference between Error and Exception?
https://stackoverflow.com/questions/5813614/what-is-difference-between-errors-and-exceptions
3. In any case, you should close the connection with DB. For this purpose, we use try-with-resources and/or finally block.
4. The same about all other methods in this class.
In this book https://careers.epam.by/content/dam/epam/by/book_epam_by/JAVA_Methods_Programming_v2.march2015.pdf
you can find how to work with JDBC driver in a more careful way.